data-raw/process_data_sets_kara.R

```diff
@@ -0,0 +1,31 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+#reading in csv
+nba_elo <- read_csv("https://projects.fivethirtyeight.com/nba-model/nba_elo.csv")
+nba_elo_latest <- read_csv("https://projects.fivethirtyeight.com/nba-model/nba_elo_latest.csv")
+
+rownames(nba_elo) <- NULL
+rownames(nba_elo_latest) <- NULL
+
+#tidying and taming data for nba_elo
+nba_elo %>% clean_names()
+gsub(x = colnames(nba_elo), pattern = "-", replacement = "_")
```

Comment:
If this ends up being necessary after the `clean_names()`, could you use `str_replace_all()` inside of a `mutate()` in the `%>%` chain instead?